### ISSUE TITLE
Display organization member roles in a matrix

### ISSUE THREAD

**Issue by s-mm** [pypi/warehouse#12431]
To improve user experience, the roles within an organization can be displayed in a matrix with check marks to indicate what each type of user can and cannot do.

The description of billing manager should also say what that role cannot do.

**Comment by ewdurbin** [comment]
So this is a proposed improvement to this?



![Image](https://user-images.githubusercontent.com/1200832/199564859-06df7468-5f59-49f0-ab29-a2df8fcad435.png)

**Comment by ewdurbin** [comment]
Seems the minimum viable resolution here would be to change the last sentence above to say "Can *only* manage billing for the organization."?

**Comment by di** [comment]
Yes, the existing paragraphs are really hard to parse for difference between the roles. Something like this would be more clear:

| Role | Member | Manager | Owner | Billing Manager |
| ----------- | ----------- | - | - | - |
| Own/maintain specific projects | ✅ |  ✅ |  ✅ |  ❌ | 
| Create new projects | ❌ |  ✅ |  ✅ |  ❌ | 
| Create/manage teams | ❌ |  ✅ |  ✅ |  ❌ | 
| Invite/manage organization members | ❌ |  ❌ |  ✅ |  ❌ | 
| Delete projects | ❌ |  ❌ |  ✅ |  ❌ | 
| Delete the organization | ❌ |  ❌ |  ✅ |  ❌ | 
| Manage billing | ✅ |  ✅ |  ✅ |  ✅ | 

(It's unclear to me who exactly has the 'manage billing' permission as well, that last line might be wrong).

> Seems the minimum viable resolution here would be to change the last sentence above to say "Can only manage billing for the organization."?

That makes it more explicit for the last one, but the challenges parsing the differences between the other roles remains.
